### ISSUE TITLE
Failed to create a account for testing. 

### ISSUE THREAD

**Issue by Ikramkhan786** [UdacityAndroidBasicsScholarship/audacity#9]
App didn't create a new account. I want to create a account for testing but it always fail.

**Comment by Suhanshu123** [comment]
There's some problem with code.. we'll implement another code soon.

**Comment by Ikramkhan786** [comment]
@Suhanshu123 means we shouldn't fork it an start working on something.

**Comment by Suhanshu123** [comment]
Yes... please wait for sometimes, I have received same issue from others
too.. I'll discuss with kshitiz suri today and update the repo

**Comment by kartikohri1712** [comment]
Check out https://github.com/UdacityAndroidBasicsScholarship/audacity/pull/11

**Comment by Hashik-Donthineni** [comment]
Why isn't the issue closed it isn't it closed and I still can't create an account?
